### PR TITLE
Update _multiselect.scss

### PR DESCRIPTION
**Pull request by eduardomart**
Fixes regression that happened at some point after this had been fixed:
https://github.com/bolt/core/pull/1899

### CODE REVIEW

File: assets/scss/modules/editor/fields/_multiselect.scss [hunk 1]
```diff
@@ -9,6 +9,7 @@
     }
 
     border: 1px dashed transparent;
+    margin-bottom: 0.4rem;
```
Comment (I-Valchev) [inline assets/scss/modules/editor/fields/_multiselect.scss:12]:
```suggestion
    &:not(:last-of-type) {
        margin-bottom: 0.4rem;
    }
```

**Comment by I-Valchev** [comment]
Ah, I think I'm the one to blame for that! https://github.com/bolt/core/pull/2283

What I wanted to do there is to remove the awkward empty space _in the white field input itself_, not between the items.

So, I wanted to get rid of this:

image

I think this will solve it (hence my suggestion to your PR):

```scss
&:not(:last-of-type) {
    margin-bottom: 0.4rem;
}
```

Do you agree @eduardomart ?

**Comment by eduardomart** [comment]
Sounds good to me 👍

**Comment by I-Valchev** [comment]
@eduardomart thanks, in that case I'm adding it to the PR. Let's see if the tests pass!

File: assets/scss/modules/editor/fields/_multiselect.scss [hunk 2]
```diff
@@ -9,6 +9,9 @@
     }
 
     border: 1px dashed transparent;
+    &:not(:last-of-type) {
+        margin-bottom: 0.4rem;
```
Comment (bobdenotter) [inline assets/scss/modules/editor/fields/_multiselect.scss:13]:
```suggestion
      margin-bottom: 0.4rem;
```

**Review by bobdenotter — APPROVED** [review]
Looks good to me! 👍

**Pull request merged**